ui/job-view/pushes/PushList.jsx

```diff
@@ -68,16 +63,48 @@ class PushList extends React.Component {
     document.title = `[${allUnclassifiedFailureCount}] ${repoName}`;
   }
 
-  getUrlRangeValues = (url) => {
-    const params = [...new URLSearchParams(url.split('?')[1]).entries()];
+  getUrlRangeValues = (search) => {
+    const params = [...new URLSearchParams(search)];
 
     return params.reduce((acc, [key, value]) => {
-      return reloadOnChangeParameters.includes(key)
+      return [
+        'repo',
+        'revision',
+        'author',
+        'startdate',
+        'enddate',
+        'nojobs',
+      ].includes(key)
```

Comment:
Copied from the `reloadOnChangeParameters` with `onchange` removed because it calls `fetchPushes` again after `this.getNextPushes` is called because it hits `componentDidUpdate`, overwriting those results.